src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -28,36 +28,22 @@
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Archives" Version="$(MicrosoftDotNetBuildTasksArchivesVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference
+      Include="$(RepoRoot)src/coreclr/tools/aot/crossgen2/crossgen2_publish.csproj"
+      AdditionalProperties="_IsPublishing=true
+        ;RuntimeIdentifier=$(PackageRID)
+        ;NativeAotSupported=$(NativeAotSupported)
+        ;CoreCLRArtifactsPath=$(CoreCLRArtifactsPath)
+        ;ObjCopyName=$(ObjCopyName)
+        ;R2ROverridePath=$(MSBuildThisFileDirectory)ReadyToRun.targets"
```

Comment:
Do we still need to pass these properties in? Can't we just set them directly in crossgen2_publish.csproj now that that project file is only responsible for publishing?

Reply:
We can probably inline some of them, but I don't think we can inline all of them. I think `_IsPublishing` needs to be set as a global property. I'll look at that in a follow-up PR.